packages/templating-runtime/templating.js

```diff
@@ -85,13 +87,13 @@ Template._applyHmrChanges = function (templateName) {
     updateTimeout = null;
 
     for (var i = 0; i < Template.__pendingReplacement.length; i++) {
-      delete Template[Template.__pendingReplacement[i]]
+      delete Template[Template.__pendingReplacement[i]];
     }
 
     Template.__pendingReplacement = [];
 
     var views = Blaze.__rootViews.slice();
-    for(var i = 0; i < views.length; i++) {
+    for (var i = 0; i < views.length; i++) {
```

Comment:
This could probably be a `forEach`

Reply:
Tried to change it but resulted in an error.

Reply:
OK, for now. We should take a look at this in detail later then.